### PR TITLE
glamorous -> emotion

### PR DESCRIPTION
**Pull request by chrisnojima**
- [x] desktop
- [x] mobile
- [x] storyshots

### CODE REVIEW

**Comment by chrisnojima** [comment]
@adamjspooner @cecileboucheron do you want to test this branch out a bit

**Comment by adamjspooner** [comment]
Desktop looks good to me. Trying out native next...

**Comment by adamjspooner** [comment]
Looks good on native too. 👍

**Comment by chrisnojima** [comment]
tnx!

@keybase/react-hackers this is ready. deprecating glamorous

File: shared/test/setupTestFrameworkScriptFile.js [hunk 1]
```diff
@@ -1,9 +1,6 @@
...
-// Used by jest. We need to force things to flush to glamor doesn't write things out of order
+// Used by jest.
 /* eslint-disable */
 // DON'T use es6 stuff here
-const {flush} = require('glamor')
 
-afterEach(() => {
-  flush()
-})
+afterEach(() => {})
```
Comment (cjb) [inline shared/test/setupTestFrameworkScriptFile.js]: Perhaps the whole file can be deleted?

**Pull request merged**